src/universal/components/NewMeetingSidebar.js

```diff
@@ -2,104 +2,83 @@
 import React from 'react';
 import {createFragmentContainer} from 'react-relay';
 import {Link} from 'react-router-dom';
-import {textOverflow} from 'universal/styles/helpers';
 import appTheme from 'universal/styles/theme/appTheme';
-import actionUIMark from 'universal/styles/theme/images/brand/mark-color.svg';
 import ui from 'universal/styles/ui';
-import makeHref from 'universal/utils/makeHref';
 import styled from 'react-emotion';
-import NewMeetingSidebarPhaseList from 'universal/components/NewMeetingSidebarPhaseList';
 
 import type {NewMeetingPhaseTypeEnum} from 'universal/types/schema.flow';
 import type {NewMeetingSidebar_viewer as Viewer} from './__generated__/NewMeetingSidebar_viewer.graphql';
-
-const BrandLogo = styled('img')({
-  display: 'block',
-  height: 'auto',
-  width: '100%'
-});
-
-const BrandBlock = styled('div')({
-  display: 'block',
-  height: 'auto',
-  left: '1rem',
-  position: 'absolute',
-  width: '1.9375rem'
-});
+import CopyShortLink from 'universal/modules/meeting/components/CopyShortLink/CopyShortLink';
+import LabelHeading from 'universal/components/LabelHeading/LabelHeading';
+import LogoBlock from 'universal/components/LogoBlock/LogoBlock';
+import NewMeetingSidebarPhaseList from 'universal/components/NewMeetingSidebarPhaseList';
 
 const Nav = styled('nav')({
   display: 'flex',
   flex: 1,
   flexDirection: 'column',
-  margin: '2rem 0 0',
   width: '100%'
 });
 
-const ShortUrl = styled('a')({
-  ...textOverflow,
-  color: appTheme.palette.dark10d,
-  display: 'block',
-  fontSize: appTheme.typography.s2,
-  lineHeight: appTheme.typography.sBase,
-  paddingRight: ui.meetingSidebarGutter,
-  textDecoration: 'none',
-  '&:hover,:focus': {
-    color: appTheme.palette.dark
-  }
-});
-
 const SidebarHeader = styled('div')({
   paddingLeft: '3.75rem',
   position: 'relative'
 });
 
 const SidebarParent = styled('div')({
-  backgroundColor: appTheme.palette.mid10l,
+  backgroundColor: ui.palette.white,
   boxShadow: 'rgba(0, 0, 0, 0.16) 0px 3px 10px, rgba(0, 0, 0, 0.23) 0px 3px 10px',
   display: 'flex',
   flexDirection: 'column',
-  padding: '2rem 0 0',
   maxWidth: ui.meetingSidebarWidth,
-  minWidth: ui.meetingSidebarWidth
+  minWidth: ui.meetingSidebarWidth,
+  padding: '1.25rem 0 0'
 });
 
 const TeamDashboardLink = styled(Link)({
-  color: appTheme.palette.cool,
+  color: ui.copyText,
   cursor: 'pointer',
-  fontFamily: appTheme.typography.serif,
   fontSize: appTheme.typography.s5,
-  fontStyle: 'italic',
-  fontWeight: 700,
+  fontWeight: 600,
   lineHeight: '1.5'
 });
 
+const SidebarSubHeading = styled('div')({
+  borderTop: `.0625rem solid ${ui.palette.light}`,
+  margin: '1.25rem 0 0 3.75rem',
+  padding: '1rem 0'
+});
+
 type Props = {
   localPhase: NewMeetingPhaseTypeEnum,
   viewer: Viewer
 }
+
 const NewMeetingSidebar = (props: Props) => {
   const {
     localPhase,
     viewer
   } = props;
   const {team: {teamId, teamName}} = viewer;
+  const relativeLink = `/retro/${teamId}`;
```

Comment:
Ditto that here, this rel link could be abstracted if that's your intention

Reply:
fixed